Makefile

```diff
@@ -62,6 +62,10 @@ nginx/cert.pem: nginx/key.pem
 secretkey:
 	openssl rand -hex 24
 
+auth:
```

Comment:
I might name this something more explicitly clear that it's for gcloud like `gcloud_auth` or something?